client/blocks/edit-gravatar/style.scss

```diff
@@ -115,6 +115,12 @@
 	}
 }
 
+.edit-gravatar.is-uploading {
+	.file-picker {
+		pointer-events: none;
```

Comment:
Alright let's go with this for now then 😄 I think it'd be nice to maybe enhance the FilePicker in a future PR to add disabled, since this seems like it'd be a common case.